### ISSUE TITLE
Photo Error Messages

### ISSUE THREAD

**Issue by tiffrobo** [zom/Zom-iOS#502]
**Case 1:** 
R sent a photo to T. T received an image URL and then an error. Error read timeout error, with no option to download. 
R tapped resend.  The photo came through 

**Case 2:** 
R sent T multiple photos. The first began as an URL but once the second photo began processing the URL went away and the first photo began processing like normal. Both photos were received with no problem. 

![img_3730](https://user-images.githubusercontent.com/30634981/34789091-835376da-f602-11e7-948a-0e048fd69421.PNG)

![img_3731](https://user-images.githubusercontent.com/30634981/34789112-922b9fca-f602-11e7-8f74-b52e6e88b291.PNG)

![img_3732](https://user-images.githubusercontent.com/30634981/34789123-9c45d688-f602-11e7-82de-b6c208aee56d.PNG)

**Case 3:** 
Error message and double checks.  R sent a photo to C. She received an error message and double checks. R never tried to resend. After some time C received the photo with no error. 

![img_96718cb17471-1](https://user-images.githubusercontent.com/30634981/34789075-6da35ec2-f602-11e7-8139-cbcaaab326e4.jpeg)

R’s phone
Device: iPhone SE
OS: 10.3.2 
App version: 114 

T's phone
Device: iPhone 7
OS: 11.2.2
App version: 114

C's phone
Device: Samsung Galaxy s7
OS: Android 6.0.1 
App version: 15.3.2 beta 1

**Comment by N-Pex** [comment]
Yes, I could repro this by having client A send client B an MP3 file and "hang" client B on a breakpoint in MessageStorage:208 for awhile, to simulate a timeout.
